deployment-examples/gcp/serverless/extension/README.md

```diff
@@ -0,0 +1,13 @@
+## Deploy as GCP Cloud Function
+
+1. For deployment to GCP Cloud Function, bundle the following files and folders into a .zip file and upload it in GCP console.
+
+- src
+- resources
+- index.js
+- package.json
+
+2. Specify the `extensionTrigger` as the entry point function in your GCP Cloud Function configuration.
+
+3. When the extension module is running as GCP Cloud Function **it will NOT create** the required resources like custom 
```

Comment:
Is it tested? Why this warning is not for notification?

Reply:
This was pre-existing documentation for the lambda function. It is required for extension module. It is not recommended only to use the notification without using extension module